packages/typescript/test/foobar/BUILD.bazel

```diff
@@ -1,6 +1,6 @@
-load("@npm_bazel_typescript//:index.bzl", "ts_library")
+load("@npm_bazel_typescript//:index.from_src.bzl", "ts_library")
 
-package(default_visibility = ["//visibility:public"])
+package(default_visibility = [])
```

Comment:
isn't this the same as no `package` declaration?

Reply:
👍 not sure how that go in there. thanks